app/src/main/java/com/poupa/vinylmusicplayer/provider/PreferencesBackedSongList.java

```diff
@@ -135,16 +136,26 @@ static SharedPreferences getPreferences() {
     static List<PreferencesBackedSongList> loadAll() {
         ArrayList<PreferencesBackedSongList> result = new ArrayList<>();
 
+        String favoritesPlaylistName = App.getStaticContext().getString(R.string.favorites);
+        PreferencesBackedSongList favoritesPlaylist = null;
+
         final SharedPreferences preferences = getPreferences();
         for (String prefName : preferences.getAll().keySet()) {
             if (prefName.startsWith(PREF_NAME_PREFIX)) {
                 final String name = prefName.substring(PREF_NAME_PREFIX.length());
+                if (name.equals(favoritesPlaylistName)){
+                    favoritesPlaylist = new PreferencesBackedSongList(name);
+                    continue;
+                }
                 result.add(new PreferencesBackedSongList(name));
             }
         }
 
         Collections.sort(result, (l1, l2) -> StringUtil.compareIgnoreAccent(l1.name, l2.name));
 
+        if (favoritesPlaylist != null)
```

Comment:
Hi, can this block moved up, in the loop, to reduce the diff
ie

```
for ... {
    if ... {
        final String name = prefName.substring(PREF_NAME_PREFIX.length());
        if (name.equals(favoritesPlaylistName)){
            result.add(0, new PreferencesBackedSongList(name));
        } else {
            result.add(new PreferencesBackedSongList(name));
        }
    }
}
```

Reply:
It won't be working. Statements we see in lines 156-157 should be added **after** line 154:
```
Collections.sort(result, (l1, l2) -> StringUtil.compareIgnoreAccent(l1.name, l2.name));
```
This way we obtain sorted list of all user's playlists. Then adding 'Favorites' on top. 

Otherwise, 'Favorites' would be sorted among others =)

Reply:
Ah ok